tests/main/snap-service-install-mode/svc.v1/meta/snap.yaml

```diff
@@ -8,6 +8,7 @@ apps:
     command: sleep infinity
     daemon: simple
     install-mode: disable
+    timer: 0:00~24:00/96 # every 15m
```

Comment:
shoudn't we create a new service instead of changing an existing one? we should test both activated and not activated services, no?

Reply:
I'll make sure to add this suggestion!

Reply:
Added a separate service to cover this!